libs/libsodium/Makefile

```diff
@@ -52,6 +52,7 @@ endef
 
 CONFIGURE_ARGS+= \
 	--disable-ssp \
+	--without-pthreads \
```

Comment:
A pull request for master should not disable features based on limitations in CC. It's meant to be used with LEDE/OpenWrt master.